tests/e2e/kitchen-sink.test.ts

```diff
@@ -88,8 +88,8 @@ beforeEach(() => {
       graphql: '^15.5.0',
       nexus: '1.1.0',
       prisma: '^3.5.0',
-      'ts-node': '^9.1.1',
-      'ts-node-dev': '^1.1.6',
+      'ts-node': '^10.8.1',
```

Comment:
I faced an types errors in the testing and fixed by upgrading the ts-node package